.ci/pytorch/common_utils.sh

```diff
@@ -149,38 +149,6 @@ function clone_pytorch_xla() {
   fi
 }
 
-function install_filelock() {
```

Comment:
This looks to be cleaning up what was already done in 72b9d45e76872f65a8de1cb1a61870739a97407a? 
Our latest commit 18848a1d64c97bb53011536ce5f8520c690189a0 seems to have accidently brought this along.  It looks good, and I can't see any other remnants of the previous commit in here.